client/styles/components/_preferences.scss

```diff
@@ -105,6 +129,59 @@
 	width: #{44 / $base-font-size}rem;
 }
 
+.preference__subheading1 {
```

Comment:
it looks like you're repeating the same code three times—is there a way to reuse the same code? maybe give the tabs the same class?